### DISCUSSION TITLE
[M-Z] (13 Giugno 2022) Esercizio 4

### DISCUSSION THREAD

**Discussion by rimaout** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#468] [Esami & Esoneri M-Z]
### Traccia

<img width="705" alt="image" src="https://github.com/user-attachments/assets/7ba7d809-5217-4383-b486-c63c5dfd54bb">

**Comment by matypist** [comment]
Segue una proposta di soluzione di Andrea Princic (@Princic-1837592), tratta dal suo file di soluzioni [20220613.pdf](https://raw.githubusercontent.com/Princic-1837592/metodi-matematici/refs/heads/main/soluzioni/20220613.pdf) nel suo [repository GitHub](https://github.com/Princic-1837592/metodi-matematici) contenente le sue soluzioni agli esami del prof. Piperno:

![image](https://github.com/user-attachments/assets/f960e44e-e85c-4934-8190-939e607357dd)
